data_pump/community.py

```diff
@@ -0,0 +1,94 @@
+import logging
+
+from utils import read_json, convert_response_to_json, do_api_post
+
+
+def import_community(metadata, group_id, handle, community_id, community2logo,
+                     imported_handle, metadatavalue, metadata_field_id, statistics):
+    """
+    Import data into database.
+    Mapped tables: community, community2community, metadatavalue, handle
+    """
+    json_name_com = 'community.json'
+    json_name_com2com = 'community2community.json'
+    url = 'core/communities'
+    importedComm = 0
+    importedGroup = 0
+    json_comm = read_json(json_name_com)
+    json_comm2comm = read_json(json_name_com2com)
+    parent = dict()
+    child = dict()
+    if json_comm2comm:
```

Comment:
Reverse the condition (use the condition bellow) and remove indentation from the body.

Reply:
it will not work